ci: build-artifact fails due the wrong flutter version

```
Run flutter build linux --release

💪 Building with sound null safety 💪

Building Linux application...                                   
ERROR: lib/components/buttons.dart:2[9](https://github.com/dart-lightning/lndart.clnapp/actions/runs/3107731556/jobs/5036161436#step:7:10):15: Error: No named parameter with the name 'backgroundColor'.
ERROR:               backgroundColor: Theme.of(context).primaryColor,
ERROR:               ^^^^^^^^^^^^^^^
ERROR: /opt/hostedtoolcache/flutter/stable-3.0.5-x64/packages/flutter/lib/src/material/elevated_button.dart:[14](https://github.com/dart-lightning/lndart.clnapp/actions/runs/3107731556/jobs/5036161436#step:7:15)6:22: Context: Found this candidate, but the arguments don't match.
ERROR:   static ButtonStyle styleFrom({
ERROR:                      ^^^^^^^^^
Build process failed
Error: Process completed with exit code 1.
```

To fix this we just remove the flutter version in https://github.com/dart-lightning/lndart.clnapp/blob/main/.github/workflows/build-binary.yml :)

Just moving from
```
      - uses: subosito/flutter-action@v2
        with:
          flutter-version: '3.0.5'
```

to

```
      - uses: subosito/flutter-action@v2
```

--- a/.github/workflows/build-binary.yml
+++ b/.github/workflows/build-binary.yml
@@ -11,9 +11,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v2
-        with:
-          flutter-version: '3.0.5'
-          channel: 'stable'
       - name: Install dependencies
         run: sudo apt-get install -y clang cmake ninja-build pkg-config libgtk-3-dev liblzma-dev
       - name: Install project dependencies
@@ -36,9 +33,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v2
-        with:
-          flutter-version: '3.0.5'
-          channel: 'stable'
       - name: Install project dependencies
         run: flutter pub get
       - name: Enable windows build
@@ -57,9 +51,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v2
-        with:
-          flutter-version: '3.0.5'
-          channel: 'stable'
       - name: Install project dependencies
         run: flutter pub get
       - name: Enable macOS build
@@ -80,9 +71,6 @@ jobs:
           with:
             java-version: '12.x'
         - uses: subosito/flutter-action@v2
-          with:
-            flutter-version: '3.0.5'
-            channel: 'stable'
         - run: flutter pub get
         - run: flutter build apk
         - uses: actions/upload-artifact@v1

